packages/components/dashboard/resource-providers-card/resource-providers-card.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  ResourceProvidersItem,
+  ProviderType,
+} from './resource-providers-card-item';
+import './resource-providers-card.scss';
+
+const RESOURCE_PROVIDERS_QUERY = {
+  PROVIDERS_TYPES: ' NooBaa_cloud_types',
+  UNHEALTHY_PROVIDERS_TYPES: 'NooBaa_unhealthy_cloud_types',
+  RESOURCES_LINK_QUERY: 'NooBaa_system_links',
+};
+
+const getProviderType = (provider: ProviderPrometheusData): string =>
+  _.get(provider, 'metric.type', null);
+const getProviderCount = (provider: ProviderPrometheusData): number =>
+  Number(_.get(provider, 'value[1]', null));
+
+const filterProviders = (allProviders: ProviderType): string[] => {
+  return _.keys(allProviders).filter((provider) => allProviders[provider] > 0);
+};
+
+const createProvidersList = (data: PrometheusResponse): ProviderType => {
+  const providers = _.get(data, 'data.result', null);
+  const providersList: ProviderType = {};
+  if (_.isNil(providers)) return {};
+  providers.forEach((provider) => {
+    providersList[getProviderType(provider)] = getProviderCount(provider);
+  });
+  return providersList;
+};
+
+const ResourceProviders: React.FC<{}> = () => {
+  const { t } = useTranslation();
+
+  const [providersTypesQueryResult, providersTypesQueryResultError] =
+    useCustomPrometheusPoll({
+      query: RESOURCE_PROVIDERS_QUERY.PROVIDERS_TYPES,
+      endpoint: 'api/v1/query' as any,
+      basePath: MCG_MS_PROMETHEUS_URL,
+    });
+  const [
+    unhealthyProvidersTypesQueryResult,
+    unhealthyProvidersTypesQueryResultError,
+  ] = useCustomPrometheusPoll({
+    query: RESOURCE_PROVIDERS_QUERY.UNHEALTHY_PROVIDERS_TYPES,
+    endpoint: 'api/v1/query' as any,
+    basePath: MCG_MS_PROMETHEUS_URL,
+  });
+  const [resourcesLinksResponse, resourcesLinksResponseError] =
+    useCustomPrometheusPoll({
+      query: RESOURCE_PROVIDERS_QUERY.RESOURCES_LINK_QUERY,
+      endpoint: 'api/v1/query' as any,
+      basePath: MCG_MS_PROMETHEUS_URL,
+    });
+  const error =
+    !!providersTypesQueryResultError ||
+    !!unhealthyProvidersTypesQueryResultError ||
+    !!resourcesLinksResponseError;
+
+  const noobaaResourcesLink = getMetric(resourcesLinksResponse, 'resources');
+
+  const allProviders = createProvidersList(providersTypesQueryResult);
+  const unhealthyProviders = createProvidersList(
+    unhealthyProvidersTypesQueryResult
+  );
+
+  const providerTypes = filterProviders(allProviders);
+  const hasProviders = !_.isEmpty(allProviders);
+  const isLoading =
+    !error &&
+    !(providersTypesQueryResult && unhealthyProvidersTypesQueryResult);
+  return (
+    <Card>
+      <CardHeader>
+        <CardTitle>{t('Resource Providers')}</CardTitle>
+      </CardHeader>
+      <CardBody>
+        {isLoading ? (
+          <Skeleton />
+        ) : error || !hasProviders ? (
+          <div className="nb-resource-providers-card__not-available text-secondary">
+            {t('Not available')}
+          </div>
```

Comment:
use ResourceProvidersBody component for this

Reply:
for `<div className="nb-resource-providers-card__not-available text-secondary">
            {t('Not available')}
          </div>`
this much code, I don't feel it is fruitful to use a new component

Reply:
not blocking this PR.